common/config/rush/common-versions.json

```diff
@@ -16,7 +14,7 @@
      * will recalculate all version selections.
      */
     "preferredVersions": {
-        "@types/node": "^16.11.11",
+        "@types/node": "^18.15.11",
```

Comment:
is this version correct since we use 16.20.0 node ?

Reply:
downgraded to latest 16.x version, thanks